config/test.exs

```diff
@@ -1,5 +1,8 @@
 import Config
 
+# Mark this as test environment for conditional logic
+config :dbservice, environment: :test
```

Comment:
This is not necessary as `test.exs` already is the test environment

Reply:
Yeah I'll try to replicate the issue by removing this piece of added code and then i can show you on call